test/integration_test/snapshot_test.go

```diff
@@ -363,7 +363,7 @@ func deletePolicyAndSnapshotSchedule(t *testing.T, namespace string, policyName
 }
 
 func intervalSnapshotScheduleTest(t *testing.T) {
-	ctx := createApp(t, "interval-snap-sched-test")
+	ctx := createApp(t, "interval-snap-sched-test", testKey)
```

Comment:
Move `testKey` to common_test.go and rename it `'defaultTestKey"

Reply:
Done